lib/specinfra/command/freebsd/v8/package.rb

```diff
@@ -0,0 +1,54 @@
+class Specinfra::Command::Freebsd::V8::Package < Specinfra::Command::Freebsd::Base::Package
+  class << self
+    def pkg_info_pattern(package)
+      # allow portorigin (origin/portname) as package argument, so that we get
+      # similar answers from either "pkg info" and "pkg_info"
+      "^#{package.split('/', 2)[-1]}-[0-9][0-9a-zA-Z_\.,]*$"
```

Comment:
This also prevents false positives when, for example, one checks for package ``foo``, which is not installed, and still gets positive answer from ``pkg_info -Ix foo`` because package ``foo_bar-1.2.3`` is installed.